canonical-webteam/global-nav
# Style small screens

## Done
Changed the styling to a dark button on small screens. Fixed the hash issue.

## QA
- Pull branch
- Run `gulp build`
- Open index.html
- On small screen check the toggle is styled dark and the it open and closes corrently
